starter/js/atm.js

```diff
@@ -1,43 +1,29 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
 
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
+//Begin with the document ready function
 
-      //On click of the withdrawl button
+$(document).ready (function() {
+    $('#depositChecking').click(function(event) {
+      var depositAmount= parseInt($('#amountChecking').val(), 10);
```

Comment:
Just FYI, the parseInt method will default to 10 as the second argument so you don't actually have to specify it. But nothing wrong with including it either!